### DISCUSSION TITLE
problem luet

### DISCUSSION THREAD

**Discussion by biandrea** [mocaccinoOS/mocaccino#146] [Q&A] [answered]
problem with luet. I come from sabayon, where I couldn't with fair I could with emerge. with mocaccino OS I have a problem with luet and I get this error
# luet install libreoffice
 INFO  Luet 0.32.5-gd5d21653d7313ff396c0511af402834c89974ee3 2022-06-07
 INFO  Copyright (C) 2019-2022 Ettore Di Giacinto
       This program comes with ABSOLUTELY NO WARRANTY.
       This is free software, and you are welcome to redistribute it under certain conditions.
       For documentation, visit https://luet.io.
 INFO   :::> Install
 INFO   🤔  Checking for available upgrades                                                                   
 INFO   Nothing to upgrade
  ERROR    Error: Failed matching solutions against repository for libreoffice where are definitions coming from?!

**Comment by joostruis** [comment, marked as answer]
Hello. Even in Sabayon Portage was not compatible with the Entropy tree (obviously).

However. You are not using Luet correctly. As explained in the [documentation](https://www.mocaccino.org/docs/desktop/usage/#installing-a-package) you need to provide the category name.

In the case of libreoffice you need to enable the community repository. Instructions for that can be found [here](https://github.com/mocaccinoOS/community-repository?tab=readme-ov-file#-community-repository).

Long story short.

First install community repository:
`sudo luet install repository/mocaccino-community-stable`

Now install libreoffice
`sudo luet install apps/libreoffice`
